src/py21cmfast/wrapper.py

```diff
@@ -1096,8 +628,8 @@ def perturb_field(
     user_params=None,
     cosmo_params=None,
     random_seed=None,
-    regenerate=False,
-    write=True,
+    regenerate=None,
+    write=None,
```

Comment:
Does this overwrite line 592?

Reply:
Yes -- these are passed through to each recursive level.